app/src/composables/use-alias-fields.ts

```diff
@@ -111,6 +111,9 @@ export function useAliasFields(
 	function getFromAliasedItem<K, T extends Record<string, K>>(item: T, key: string): K | undefined {
 		const aliasInfo = Object.values(aliasedFields.value).find((field) => field.key === key);
 
+		// Skip any thumbnail keys as they don't actually exist
+		key = key.replace('.$thumbnail', '');
```

Comment:
Can we do it for any key starting with `$` instead? `$thumbnail` is the only one currently, but i'd like it to be as generic as possible